docs/userguide.md

```diff
@@ -251,7 +251,7 @@ kubectl -n test-namespace edit federatednamespaceplacement test-namespace
 Then wait to verify all resources are removed from `cluster2`:
 
 ```bash
-for r in configmaps secrets service deploy; do
+for r in configmaps secrets service deploy rs job; do
```

Comment:
I'm not sure that adding replicasets to the example really adds that much value as we already have deployments. It becomes a little repetitive and requires even more resources to run the example. We could just add a note about them, or list/point to the set of resources we support. I'm okay with jobs though.

Reply:
Done, deleted rs.